Wikipedia/Code/WMFArticleViewController.m

```diff
@@ -1164,6 +1164,9 @@ - (void)fetchArticleForce:(BOOL)force {
                                                dismissPreviousAlerts:NO
                                                          tapCallBack:NULL];
                 }
+            } else if ([error wmf_isWMFErrorMissingTitle]) {
+                [self wmf_openExternalUrl:self.articleURL];
+                [self.navigationController popToRootViewControllerAnimated:YES];
```

Comment:
Probably just want to pop the current VC instead of popping to root. If the user is more than one article deep on the nav stack, they should only go back to the previous article, not all the way back to explore, saved, history, or places.